github/Requester.py

```diff
@@ -426,7 +426,7 @@ def requestJsonAndCheck(
         parameters: Optional[Dict[str, Any]] = None,
         headers: Optional[Dict[str, str]] = None,
         input: Optional[Any] = None,
-    ) -> Tuple[Dict[str, Any], Optional[Dict[str, Any]]]:
```

Comment:
Why removing the `Dict` here?

Reply:
It parse request response body, not necessarily dict